flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip-icons-check": true,
+    "only-arches": ["x86_64"]
```

Comment:
I guess it actually can work on non-x86_64?

Reply:
See [this comment](https://github.com/flathub/flathub/pull/3337#issuecomment-1189454439).